[2025-06-12T01:45:45.127+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dynamic_training_pipeline.merge_data scheduled__2024-12-05T15:00:00+00:00 [queued]>
[2025-06-12T01:45:45.145+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dynamic_training_pipeline.merge_data scheduled__2024-12-05T15:00:00+00:00 [queued]>
[2025-06-12T01:45:45.146+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-06-12T01:45:45.157+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): merge_data> on 2024-12-05 15:00:00+00:00
[2025-06-12T01:45:45.162+0000] {standard_task_runner.py:60} INFO - Started process 629 to run task
[2025-06-12T01:45:45.172+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dynamic_training_pipeline', 'merge_data', 'scheduled__2024-12-05T15:00:00+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/dag_dynamic.py', '--cfg-path', '/tmp/tmp8oof53ss']
[2025-06-12T01:45:45.176+0000] {standard_task_runner.py:88} INFO - Job 33: Subtask merge_data
[2025-06-12T01:45:45.248+0000] {task_command.py:423} INFO - Running <TaskInstance: dynamic_training_pipeline.merge_data scheduled__2024-12-05T15:00:00+00:00 [running]> on host caffbd336f08
[2025-06-12T01:45:45.302+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dynamic_training_pipeline' AIRFLOW_CTX_TASK_ID='merge_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-05T15:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-05T15:00:00+00:00'
[2025-06-12T01:45:45.304+0000] {logging_mixin.py:188} INFO - Archivo no encontrado en raíz: /opt/***/data_1.csv
[2025-06-12T01:45:45.304+0000] {logging_mixin.py:188} INFO - Archivo no encontrado en raíz: /opt/***/data_2.csv
[2025-06-12T01:45:45.305+0000] {logging_mixin.py:188} INFO - No hay datos para concatenar.
[2025-06-12T01:45:45.305+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-12T01:45:45.314+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dynamic_training_pipeline, task_id=merge_data, execution_date=20241205T150000, start_date=20250612T014545, end_date=20250612T014545
[2025-06-12T01:45:45.354+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-12T01:45:45.371+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
